test/spec/CodeEditor.spec.js

```diff
@@ -692,8 +704,13 @@ return
       await expectEventually(() => {
         const completions = currentCompletions(cm.state);
 
-        expect(completions).not.to.be.empty;
-        expect(completions[0].label).to.have.eql('abs(n)');
+        expect(completions).to.have.length.gte(2);
+
+        // variable completions offered
+        expect(completions[0].label).to.have.eql('ab');
+
+        // followed by function completions
+        expect(completions[1].label).to.have.eql('abs(n)');
```

Comment:
```suggestion
        // variable completions offered
        expect(completions[0].label).to.eql('ab');

        // followed by function completions
        expect(completions[1].label).to.eql('abs(n)');
```

Reply:
Thanks! Found and fixed additional instances -https://github.com/bpmn-io/feel-editor/pull/56/commits/8ae2242e408b8dadc68290d95ef1010bf2ea0b0d.